app/services/support_interface/candidate_journey_tracker.rb

```diff
@@ -141,9 +151,16 @@ def received_references
       @received_references ||= all_references.select(&:feedback_provided?)
     end
 
-    def received_reference_times
+    def received_references_timings
       @received_reference_times ||=
-        received_references.map { |reference| earliest_update_audit_for(reference, feedback_status: 'feedback_provided') }.compact.sort
+        received_references.map do |reference|
+          {
+            received_at: earliest_update_audit_for(reference, feedback_status: 'feedback_provided'),
+            requested_at: reference.requested_at,
+          }
+        end
```

Comment:
Nice :ok_hand: . Had a mind to make the references available in their own private method, but it's more than clear enough what's going on here.